src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Checkout/AddressOrderHandlerSpec.php

```diff
@@ -187,6 +195,50 @@ function it_handles_addressing_an_order_for_not_logged_in_shop_user(
         $this($addressOrder);
     }
 
+    function it_does_not_create_new_addresses_for_already_addressed_order(
```

Comment:
This is not 100% true, you are creating new addresses, but also removing the old ones.